pandas/tests/io/test_iceberg.py

```diff
@@ -0,0 +1,143 @@
+"""
+Tests for the Apache Iceberg format.
+
+Tests in this file use a simple Iceberg catalog based on SQLite, with the same
+data used for Parquet tests (``pandas/tests/io/data/parquet/simple.parquet``).
+"""
+
+import collections
+import importlib
+import pathlib
+
+import pytest
+
+import pandas as pd
+import pandas._testing as tm
+
+from pandas.io.iceberg import read_iceberg
+
+pytestmark = [pytest.mark.single_cpu]
```

Comment:
```suggestion
pytestmark = pytest.mark.single_cpu
```

Nit: Since there only 1 entry